script.js

```diff
@@ -89,3 +89,238 @@ form.addEventListener("submit", (event) => {
+
+  const projectTitle = document.createElement('h2');
+  projectTitle.className = 'project-heading';
+  projectTitle.textContent = project.title;
+  projectContent.appendChild(projectTitle);
+
+  const projectLinks = document.createElement('div');
+  projectLinks.className = 'project-links';
+  projectContent.appendChild(projectLinks);
+
+  const liveLink = document.createElement('a');
+  liveLink.href = project.liveLink;
+  liveLink.textContent = 'See Live';
+  projectLinks.appendChild(liveLink);
+
+  const liveImg = document.createElement('img');
+  liveImg.className = 'live-img';
+  liveImg.src = './assets/SeeLive.png';
+  liveImg.alt = 'live';
+  liveLink.appendChild(liveImg);
+
+  const githubLink = document.createElement('a');
+  githubLink.href = project.githubLink;
+  githubLink.textContent = 'See Source';
+  projectLinks.appendChild(githubLink);
+
+  const githubImg = document.createElement('img');
+  githubImg.className = 'github-img';
+  githubImg.src = './assets/SeeSource-GitHub.png';
+  githubImg.alt = 'github';
+  githubLink.appendChild(githubImg);
+
+  const technologiesList = document.createElement('ul');
+  technologiesList.className = 'technologies-list';
+  project.technologies.forEach((technology) => {
+    const technologyItem = document.createElement('li');
+    technologyItem.textContent = technology;
+    technologiesList.appendChild(technologyItem);
+  });
+  popupWindow.appendChild(technologiesList);
+
+  const projectDescription = document.createElement('p');
+  projectDescription.className = 'project-description';
+  projectDescription.textContent = project.description;
+  popupWindow.appendChild(projectDescription);
+
+  popup.appendChild(popupWindow);
+  document.body.appendChild(popup);
+}
+
+projects.forEach((project) => {
+  const projectTile = document.createElement('div');
+  projectTile.className = 'project-tile';
+
+  const image = document.createElement('img');
+  image.className = 'project-image';
+  image.src = project.featuredImage;
+  image.alt = project.title;
+  projectTile.appendChild(image);
+
+  const title = document.createElement('h3');
+  title.className = 'project-title';
+  title.textContent = project.title;
+  projectTile.appendChild(title);
+
+  const technologiesList = document.createElement('ul');
+  technologiesList.className = 'tech-list';
+  project.tech.forEach((technology) => {
+    const technologyItem = document.createElement('li');
+    technologyItem.className = 'tech-item';
+    technologyItem.textContent = technology;
+    technologiesList.appendChild(technologyItem);
+  });
+  projectTile.appendChild(technologiesList);
+
+  const seeProjectButton = document.createElement('button');
+  seeProjectButton.className = 'see-project btn';
+  seeProjectButton.textContent = 'See Project';
+  projectTile.appendChild(seeProjectButton);
+
+  seeProjectButton.addEventListener('click', () => {
+    showProjectDetails(project);
+  });
+
+  projectSection.appendChild(projectTile);
+});
```

Comment:
Nice job implementing some of the project requirements in the Figma design, Please implement the following changes ,

1.  the Popup for your design doesn't include a close button as it it shown in the Figma design. Please include this feature so that we can be able to close a popup window when it appears 😄 

Your design :
![image](https://github.com/Peter-Kibet/Peter-K-Portfolio/assets/20537712/16a8be83-45fe-4531-8596-04a0f78f0997)

Figma design :
![image](https://github.com/Peter-Kibet/Peter-K-Portfolio/assets/20537712/c1dd9d5b-d64e-4d27-8512-ddd722efac3c)

2. On the mobile design, the popup window is not exactly the design we have for the template in your figma design , please ensure to use the right format

Your design:
![image](https://github.com/Peter-Kibet/Peter-K-Portfolio/assets/20537712/8f322560-6770-4199-a23b-f7f418770a1b)

Figma design :
![image](https://github.com/Peter-Kibet/Peter-K-Portfolio/assets/20537712/579b3d49-3dbd-40c3-a1a4-38da6df39ad8)